addons/website_event/controllers/main.py

```diff
@@ -309,7 +381,10 @@ def _process_attendees_form(self, event, form_details):
                 registration_index, field_name = key_values
                 if field_name not in registration_fields:
                     continue
-                registrations.setdefault(registration_index, dict())[field_name] = int(value) or False
+                reg = registrations.setdefault(registration_index, dict())
+                reg[field_name] = int(value) or False
+                if form_details.get('selected_slot'):
+                    reg['event_slot_id'] = int(form_details['selected_slot'])
```

Comment:
Just to get it right, here we handle fields that are 'generic', not linked to a specific registration, right?